src/Algebra/Definitions.agda

```diff
@@ -161,3 +161,15 @@ LeftDivides ∙ \\ = (LeftDividesˡ ∙ \\) × (LeftDividesʳ ∙ \\)
 
 RightDivides : Op₂ A → Op₂ A → Set _
 RightDivides ∙ // = (RightDividesˡ ∙ //) × (RightDividesʳ ∙ //)
+
+StarMakeRight : A → Op₂ A → Op₂ A → Op₁ A → Set _
```

Comment:
`Make` is a bad name, IMHO. Perhaps `StarRightExpansion` ? Or `KleeneStarRightExpansion` ? (Same comment for `Left`)

Reply:
We can go with `StarRightExpansion`. `StarMakeRight` was taken from a paper.